src/lib/azure/deploymenttable.ts

```diff
@@ -32,6 +34,139 @@ export interface DeploymentEntry {
   pr?: string;
 }
 
+export interface DeploymentRow {
+  status?: string;
+  service?: string;
+  env?: string;
+  author?: string;
+  imageTag?: string;
+  srcToAcrPipelineId?: string;
+  srcToAcrResult?: string;
+  srctoAcrCommitId?: string;
+  AcrToHldPipelineId?: string;
+  AcrToHldResult?: string;
+  AcrToHldCommitId?: string;
+  HldToManifestPipelineId?: string;
+  HldToManifestResult?: string;
+  HldToManifestCommitId?: string;
+  pr?: string;
+  mergedBy?: string;
+  duration?: string;
+  endTime?: string;
+  syncStatus?: string;
+}
+
+/**
+ * Prints deployment table
+ * @param outputFormat output format: json, wide, normal
+ * @param deployments list of deployments to print
+ * @param removeSeparators Whether to remove separators or not
+ */
+export const printDeploymentTable = (
+  outputFormat: OUTPUT_FORMAT,
+  deployments: DeploymentRow[],
+  removeSeparators?: boolean
+): Table => {
+  let header = ["Status", "Service", "Ring"];
+
+  if (outputFormat === OUTPUT_FORMAT.WIDE) {
+    header = header.concat(["Author"]);
```

Comment:
can be just `header.push("Author")` so we do not need to construct a new array object